src/Compilers/VisualBasic/Portable/Binding/Binder_Initializers.vb

```diff
@@ -30,12 +30,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End Get
                 Set(value As ImmutableArray(Of BoundStatement))
                     Debug.Assert(Not value.IsDefault)
-                    ImmutableInterlocked.InterlockedInitialize(_loweredInitializers, value)
+                    Debug.Assert(_loweredInitializers.IsDefault)
```

Comment:
I must be missing something obvious.  Isn't this set to Empty in the constructor?